priam/src/main/java/com/netflix/priam/aws/S3BackupPath.java

```diff
@@ -86,7 +86,7 @@ public void parseRemote(String remoteFilePath) {
         token = pieces.get(3);
         time = parseDate(pieces.get(4));
         type = BackupFileType.valueOf(pieces.get(5));
-        if (type != BackupFileType.META && type != BackupFileType.CL) {
+        if (type != BackupFileType.META && type != BackupFileType.META_V2 && type != BackupFileType.CL) {
```

Comment:
nit: extract this into a method (maybe isDataFile()) since this is used in multiple places

Reply:
Done